packages/swap/src/helpers/orders/getBuyAmount.ts

```diff
@@ -30,8 +30,8 @@ export const getBuyAmount = (
     return {
       tokenId,
       quantity: asQuantity(
-        BigNumber(sell.quantity)
-          .dividedToIntegerBy(BigNumber(limit))
+        new BigNumber(sell.quantity)
```

Comment:
From their docs, `new is optional`
```
x = new BigNumber(123.4567)                // '123.4567'
// 'new' is optional
y = BigNumber(x)                           // '123.4567'
```

But we can add them if you like them

Reply:
TS complains after BN 9.1

Reply:
Uh, ok, I hadn't noticed anything but probably didn't do any install yet since 9.1 4 days ago